README.md

```diff
@@ -128,15 +206,17 @@ $ sudo make install
 $ make BUILDTAGS="seccomp noembed"
 ```
 
-#### Alpine Linux
+#### Linux Distribution Packages
+
+##### Alpine Linux
 
 There is an [APKBUILD](https://pkgs.alpinelinux.org/package/edge/testing/x86_64/img).
 
 ```console
 $ apk add img --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
 ```
 
-#### Arch Linux
+##### Arch Linux
```

Comment:
Hey @kekoav! Could double-check on this section, please? I believe I ran into a a mistake on [line 228](https://github.com/genuinetools/img/pull/287/commits/898e3f9f021495718fe6288e97b94692ef722c16#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R228):

```markdown
- $ git clone https://aur.archlinux.org/packages/img.git
+ $ git clone https://aur.archlinux.org/img.git
```
The current link gives me this:

```console
λ git clone https://aur.archlinux.org/packages/img.git
Cloning into 'img'...
fatal: repository 'https://aur.archlinux.org/packages/img.git/' not found
```

The suggested one (from [aur.archlinux.org](https://aur.archlinux.org/packages/img/)) works fine:

```console
λ git clone https://aur.archlinux.org/img.git
Cloning into 'img'...
remote: Enumerating objects: 13, done.
remote: Counting objects: 100% (13/13), done.
remote: Compressing objects: 100% (11/11), done.
remote: Total 13 (delta 1), reused 13 (delta 1), pack-reused 0
Unpacking objects: 100% (13/13), 2.98 KiB | 2.98 MiB/s, done.
```

In case you think this fix is out of the scope of your PR, LMK and I'll gladly open one with the fix :smile: 

BTW, great docs! Looking forward to the section with examples for GitLab CI! I'm more than willing to help with that in case you need. I'm currently going through [some trouble](https://github.com/genuinetools/img/issues/228) and this section would help my team and I a ton! :tada: